### ISSUE TITLE
Update Dependencies to include React 16.0.0

### ISSUE THREAD

**Issue by gtwilliams03** [schneidmaster/react-bootstrap-maskedinput#10]
I am getting an error in my project now that `Uncaught Error: Element ref was specified as a string (inner) but no owner was set. You may have multiple copies of React loaded.` 

I think this is because this, and another package are loading React 15 - is it possible to simply make `react` a peer dependency?

```
gtw$ yarn list react
yarn list v1.2.0
├─ react-bootstrap-maskedinput@0.3.0
│  └─ react@15.6.2
├─ react@16.0.0
└─ theming@1.1.0
   └─ react@15.6.2
✨  Done in 1.28s.
```

**Comment by schneidmaster** [comment]
Fixed in 0.4.0. (`react-bootstrap-maskedinput` has not released a version that lists React 16 in its dependencies yet, so you'll still get a warning for now, though as far as I can tell it works fine.)

**Comment by gtwilliams03** [comment]
Thank you!
